articles/container-instances/container-instances-container-groups.md

```diff
@@ -61,15 +61,21 @@ In this scenario, you could set a resource limit of 2 CPUs for the instance. Thi
 
 ## Networking
 
-Container groups can share an external-facing IP address and a port namespace on that IP address. To enable external clients to reach a container within the group, you must expose the port on the IP address and from the container. Because containers within the group share a port namespace, port mapping isn't supported. 
+Container groups can share an external-facing IP address, a port on that IP address, and a DNS label with a fully qualified domain name (FQDN). To enable external clients to reach a container within the group, you must expose the port on the IP address and from the container. Because containers within the group share a port namespace, port mapping isn't supported. A container group's IP address and FQDN will be released when the container group is deleted. 
```

Comment:
Suggest changing "a port on that IP address" to "**one or more ports** on that IP address" - b/c the API supports a list of ports on container groups. Otherwise LGTM!